nni/retiarii/nn/pytorch/mutator.py

```diff
@@ -234,9 +283,10 @@ def extract_mutation_from_pt_module(pytorch_model: nn.Module) -> Tuple[Model, Op
         # tricky case: value choice that serves as parameters are stored in traced arguments
         if is_basic_unit(module):
             for key, value in module.trace_kwargs.items():
-                if isinstance(value, ValueChoice):
-                    node = graph.add_node(name + '.init.' + key, 'ValueChoice', {'candidates': value.candidates})
-                    node.label = value.label
+                if isinstance(value, ValueChoiceX):
+                    for i, choice in enumerate(value.inner_choices()):
+                        node = graph.add_node(f'{name}.init.{key}.{i}', 'ValueChoice', {'candidates': choice.candidates})
+                        node.label = choice.label
```

Comment:
`extract_mutation_from_pt_module` is for python engine, right? the returned model of this function is not a connected graph, right?

Reply:
Yes. The nodes only exist to remind the sampler that they need to be sampled.